### PR TITLE
A sample that shows how to serve static via nginx on app engine flex.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

**Comment by bshaffer** [comment]
LGTM!

File: appengine/flexible/staticcontent/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Static content and Google App Engine Flexible Environment
+
+This sample application demonstrates how to serve static content with Google App
```
Comment (JustinBeckwith) [inline appengine/flexible/staticcontent/README.md]: This is a total nit - but can we be very specific here that this is for PHP only?  PHP is the only runtime with nginx baked into the app container.

**Comment by JustinBeckwith** [comment]
LGTM

File: appengine/flexible/staticcontent/README.md (hunk 1 above)
Comment (SurferJeffAtGoogle) [inline appengine/flexible/staticcontent/README.md]: I agree this is an important fact for users to consider.  Do you think Python programmers may stumble into the php-docs-samples repo and come to the wrong conclusion?

**Pull request merged**